Assets/HoloToolkit-Examples/HeadsUpDirectionIndicator/Example.unity

```diff
@@ -0,0 +1,431 @@
+%YAML 1.1
```

Comment:
Thanks @NeerajW. I was following some recs from other feedback but I think you're right. These should be public. That makes it so the indicator is scriptable. That does mean the null check will have to stay in Update in case targetObject is reassigned.